DATATSTD-2371 New values to Group 2 descriptors

mragonech opened the pull request:
DATATSTD-2371 New values to Group 2 descriptors per CEDS alignment

mragonech committed:
DATATSTD-2371 New values to Group 2 descriptors

DATATSTD-2371 New values to Group 2 descriptors per CEDS alignment

mmarable-dlp reviewed (changes requested):
See comments in ticket DATASTD-2371 for changes requested

mragonech committed:
DATASTD-2371 Semi-Alphabetized descriptors

DATASTD-2371 Semi-Alphabetized descriptors, added Shotgun.

mmarable-dlp reviewed (changes requested):
please see DATASTD-2371 for additional request changes

mragonech committed:
DataSTD-2371 reduced value - added description

DataSTD-2371 reduced value - added description - incorporate changes from review.

mragonech committed:
DATASTD-2371 fixed Weapon Descriptor

DATASTD-2371 fixed Weapon Descriptor
